### ISSUE TITLE
Command to list outdated packages

### ISSUE THREAD

**Issue by Hurtak** [elm-lang/elm-package#218]
Npm currently provides `npm oudated` command which lists versions of currently installed packages, versions specifiend in package.json and latest avaliable version of packages.

Output of that command looks like this:
```
Package                    Current   Wanted   Latest  Location
babel-core                   6.7.4    6.7.4    6.9.0  project-name
babel-polyfill               6.7.4    6.7.4    6.9.0  project-name
babel-preset-es2015          6.6.0    6.6.0    6.9.0  project-name
```

__Proposal__: add `elm package outdated` which will show similar output to the `npm outdated` command

**Comment by smorele** [comment]
Good idea I think can be included in https://github.com/elm-lang/elm-package/issues/220

**Comment by romariolopezc** [comment]
@Hurtak what do you think about this? https://github.com/gyzerok/elm-outdated

**Comment by Hurtak** [comment]
@romariolopezc Looks swell :)
